### DISCUSSION TITLE
Background image for certain trials

### DISCUSSION THREAD

**Discussion by einGlasRotwein** [jspsych/jsPsych#1311] [Q&A]
Hello everyone,

how do I set  background image for a certain trial type, say, only for the fixation screen.  I was able to set a background image for the whole task based on [this discussion](https://github.com/jspsych/jsPsych/discussions/1133).  I am also able to change the background colour, e.g. during `on_finish()` of the previous trial using `document.body.style.background = '#000000';` or, alternatively:

```
disp = document.querySelector('.jspsych-display-element');
disp.style.background = '#000000';
```

However, this does not work for images. If I use the image path (`document.body.style.backgroundImage = "./images/example.png";`), nothing happens at all.  If I use `url()` on the path (which works fine if I use it as style in the body like in the discussion linked above; `document.body.style.backgroundImage = url("./images/example.png");`), I get an error that the url is not defined.

How do I change the background image dynamically?

Thank you in advance
Juli

**Comment by becky-gilbert** [comment]
Hi Juli! Your second solution is correct, you just need to put the `url()` in quotes:
```javascript
document.body.style.backgroundImage = 'url("./images/example.png")';
```
See [here for the docs on setting style.backgroundImage](https://www.w3schools.com/jsref/prop_style_backgroundimage.asp). I hope that helps!
